geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/data/AccumuloDataStoreFactory.scala

```diff
@@ -138,23 +137,22 @@ object AccumuloDataStoreFactory {
   }
 
   def buildConfig(connector: Connector, params: JMap[String, Serializable]): AccumuloDataStoreConfig = {
-    val catalog = tableNameParam.lookUp(params).asInstanceOf[String]
+    val catalog = CatalogParam.lookup(params)
 
     val authProvider = buildAuthsProvider(connector, params)
     val auditProvider = buildAuditProvider(params)
 
-    val auditQueries = !connector.isInstanceOf[MockConnector] &&
-        (auditQueriesParam.lookupWithDefault[Boolean](params) || collectQueryStatsParam.lookupWithDefault[Boolean](params))
+    val auditQueries = !connector.isInstanceOf[MockConnector] && AuditQueriesParam.lookup(params)
```

Comment:
Is there a logic change here?  I don't see the collectQueryStatsParam mentioned?

Reply:
collectQueryStats was redundant with audit queries, so I merged the two.

Reply:
+1